whitepapers/0013_ccq.md

```diff
@@ -404,7 +404,7 @@ uint32  response_len
    - The `slot_number` is the slot number returned by the query.
    - The `block_time_us` is the timestamp of the block associated with the slot.
    - The `block_hash` is the block hash associated with the slot.
-   - The array of results returns the data for each account queried.
+   - The array of results array returns the data for each account queried.
```

Comment:
Sorry I meant:

> The `results` array returns the data for each account queried

Just to keep with the theme of backticking the elements of the body struct

Reply:
Gotcha. Will fix this in the next PR.